oat-sa/extension-tao-itemqti
# Feature/tao 10496/tao 10501/tao 10502 per interaction response processing

Related to : 
- https://oat-sa.atlassian.net/browse/TAO-10496
- https://oat-sa.atlassian.net/browse/TAO-10501
- https://oat-sa.atlassian.net/browse/TAO-10502
 
- add configuration for per interaction response processing
- create interaction response outcome declarations when new interaction added to item
- rename interaction response outcome when response identifier changed
  
#### How to test
 
- prepare an instance of TAO
- create new test item 
- open authoring test item
- try to add some interaction
- edit response for this interaction
- check that response outcome declaration was added
- try to change response identifier
- check that outcome declaration was renamed
  
#### Dependencies
 
 - [ ] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/106
